source/guides/guides/parallelization.md

```diff
@@ -2,45 +2,74 @@
 title: Parallelization
 ---
 
-{% note warning %}
-This documentation is still a work in progress.
+{% note info %}
+
+# {% fa fa-graduation-cap %} What You'll Learn
+
+- How to parallelize test runs
+- How to group test runs
+- Strategies for grouping test runs
+- How test spec load-balancing works
+- Gaining better insights into tests with Dashboard spec views
```

Comment:
Maybe "To gain better insights into tests..." or something that makes sense in the terms of "You'll learn..."